README.md

```diff
@@ -46,108 +65,171 @@ noobaa install
+```bash
 $ noobaa version
+```
 
+```
 INFO[0000] CLI version: 5.15.0
 INFO[0000] noobaa-image: noobaa/noobaa-core:5.15.0
 INFO[0000] operator-image: noobaa/noobaa-operator:5.15.0
-
 ```
 
-# Troubleshooting
-
+## Troubleshooting
 - Verify that there are enough resources for noobaa pods:
     - `kubectl describe pod | less`
     - `kubectl get events --sort-by .metadata.creationTimestamp`
 - Make sure that there is a single **default** storage class:
     - `kubectl get sc`
     - or specify which storage class to use with `noobaa install --db-storage-class XXX --pv-pool-default-storage-class YYY`
 
-# Documentation
-- [About NooBaa](doc/about-noobaa.md)
-- CRDs
-    - [NooBaa](doc/noobaa-crd.md) - The basic CRD to deploy a NooBaa system.
-    - [BackingStore](doc/backing-store-crd.md) - Storage resources.
-    - [NamespaceStore](doc/namespace-store-crd.md) - Data resources.
-    - [BucketClass](doc/bucket-class-crd.md) - Policies applied to a class of buckets.
-- [OBC Provisioner](doc/obc-provisioner.md) - Method to claim a new/existing bucket.
-
-# Developing
-
-- Fork and clone the repo: `git clone https://github.com/<username>/noobaa-operator`
-- Use minikube: `minikube start`
+## Documentation
+1) You can find documentation related to noobaa operator and noobaa components in kubernetes in [doc](doc) directory.
+For example:
+- [NooBaa](doc/about-noobaa.md) - Basic terminology and links to videos.
+- Customer Resources Definitions ([CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#adding-custom-resources)):
+A custom resource is an extension of the Kubernetes API that is not necessarily available in a default Kubernetes installation. CRDs allow users to create new types of resources without adding another API server.
+Once a custom resource is installed, users can create, update and access its objects using [`kubectl`](https://kubernetes.io/docs/reference/kubectl/), just as they do for built-in resources.
+  - [NooBaaSystem](doc/noobaa-crd.md) - The basic CRD to deploy a NooBaa system. Represents a single installation of NooBaa that includes a set of sub-resources (backing-stores, bucket-classes, and buckets).
+  - [BackingStore](doc/backing-store-crd.md) - Storage resources. These storage targets are used to store deduplicated, compressed and encrypted chunks of data.
+  - [NamespaceStore](doc/namespace-store-crd.md) - Data resources. These storage targets are used to store and read plain data.
+  - [BucketClass](doc/bucket-class-crd.md) - Policies applied to a class of buckets, defines bucket policies relating to data placement.
+  - [Account](doc/noobaa-account-crd.md) - We use the account to receive new credentials set for accessing different noobaa services.
+- Bucket Claim:
+  - [OBC Provisioner](doc/obc-provisioner.md) - Although it using a library that is not active, we still have this option in our system.
+  - [COSI Provisioner](doc/obc-cosi-provisioner.md) - Using an active library.
```

Comment:
fix link. also fix description - COSI is a new kubernetes storage standard (like CSI) to provision object storage buckets.

Reply:
Changed, thanks.